packages/core/src/ActiveWorkflows.ts

```diff
@@ -104,6 +98,8 @@ export class ActiveWorkflows {
 			}
 		}
 
+		this.activeWorkflows[workflowId] = { triggerResponses };
```

Comment:
Since this changes the behaviour and potentially fixes a bug in the activation, should we change the PR title to `fix(core)` so it's included in the changelog? Or we could also extract this to a separate PR

Reply:
Updated the title. I don't think a separate PR is necessary here.